dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -201,6 +201,9 @@ responseType.token=Access token
 
 responseMode.formPost=Form post
 
+useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
+useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the OpenID Connect client creates HTTP client connections. Set this property to true if you want the connections to use the http* or javax* system properties.
```

Comment:
Only nit on this is that technically only the `oidcLogin` and `googleLogin` elements are Open ID Connect clients; the other social login providers are considered OAuth clients, so saying "when the OpenID Connect client creates HTTP client connections" isn't entirely accurate.